tuf/init.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/flynn/go-docopt"
+	"github.com/flynn/go-tuf"
+)
+
+func init() {
+	register("init", cmdInit, `
+usage: tuf init [--consistent-snapshot=false]
```

Comment:
Should this default to `true`? Despite the docs I expect people will call it accidentally/redundantly.